### PR TITLE
fix bug of field options of contentType

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Controller/ContentTypeController.php [hunk 1]
```diff
@@ -77,22 +77,9 @@ public function newAction(Request $request)
...
-            $this->get('session')->getFlashBag()->add(
-                'success',
-                $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation')
-            );
-
+        if (!$request->get('no_save')) {
+            $this->handleForm($form, $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation'), $contentType);
             $this->dispatchEvent(ContentTypeEvents::CONTENT_TYPE_CREATE, new ContentTypeEvent($contentType));
-
-            return $this->redirect(
```
Comment (nicolasThal) [inline BackofficeBundle/Controller/ContentTypeController.php]: it think that if the form is valid and there is no no_save parameter you should still do the redirect response

**Comment by itkg-ngilain** [comment]
I let @itkg-alavieille answer to the @nicolasThal 's remark but it's ok for me if it works as described
:+1:

File: BackofficeBundle/Controller/ContentTypeController.php (hunk 1 above)
Comment (itkg-alavieille) [inline BackofficeBundle/Controller/ContentTypeController.php]: Why redirect to the edit form ?

Comment (nicolasThal) [inline BackofficeBundle/Controller/ContentTypeController.php]: If you save the content-type, it is more userfriendly that you can directly edit it without going on an other page.

It is the workflow set on all the others cruds

Comment (itkg-alavieille) [inline BackofficeBundle/Controller/ContentTypeController.php]: But this method is called with Ajax so if i make a redirect , it's not visible by user

**Comment by itkg-alavieille** [comment]
@nicolasThal

File: BackofficeBundle/Controller/ContentTypeController.php [hunk 2]
```diff
@@ -77,22 +77,14 @@ public function newAction(Request $request)
...
-                'success',
-                $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation')
-            );
-
-            $this->dispatchEvent(ContentTypeEvents::CONTENT_TYPE_CREATE, new ContentTypeEvent($contentType));
-
-            return $this->redirect(
-                $this->generateUrl('open_orchestra_backoffice_content_type_form', array('contentTypeId' => $contentType->getContentTypeId()))
-            );
+                return $this->redirect($this->generateUrl('open_orchestra_backoffice_content_type_form', array(
```
Comment (nicolasThal) [inline BackofficeBundle/Controller/ContentTypeController.php]: Befor the redirection, can you check if the contentType was valid and saved in the database (either you check the form, or if the mongo id ofthe contentType is set)

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**